src/lavinmq/amqp/client.cr

```diff
@@ -510,8 +511,8 @@ module LavinMQ
           redeclare_exchange(e, frame)
         elsif frame.passive
           send_not_found(frame, "Exchange '#{frame.exchange_name}' doesn't exists")
-        elsif frame.exchange_name.starts_with? "amq."
-          send_access_refused(frame, "Not allowed to use the amq. prefix")
+        elsif PrefixValidation.invalid?(frame.exchange_name)
```

Comment:
I wnt to gt the prefix passed here, so we can have better log msgs and write out the prefix instead of "that"

Reply:
Or log "Prefix forbidden"?